packages/cozy-stack-client/src/ShortcutsCollection.js

```diff
@@ -14,16 +15,32 @@ class ShortcutsCollection extends DocumentCollection {
    * @param {string} attributes.name Filename
    * @param {string} attributes.url Shortcut's URL
    * @param {string} attributes.dir_id dir_id where to create the shortcut
+   * @throws {Error} - explaining reason why creation failed
    */
   async create(attributes) {
     if (!attributes.type) {
       attributes.type = SHORTCUTS_DOCTYPE
     }
-    if (!attributes.name || !attributes.url || !attributes.dir_id) {
+    if (
+      !attributes.name ||
+      !attributes.name.trim() ||
```

Comment:
why this specific line?

Reply:
When name was "    " , we used to send no error.

Reply:
https://github.com/cozy/cozy-client/pull/1085/files#diff-4d747c875bb3490e6493ca37186847024c987742d081ad7be03b86676097241cR82